spec/scripts/send_pms_spec.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+require_relative '../discourse_automation_helper'
+
+describe 'SendPms' do
+  fab!(:automation) { Fabricate(:automation, script: DiscourseAutomation::Scriptable::SEND_PMS, trigger: 'stalled_wiki') }
+
+  before do
+    automation.upsert_field!('sender', 'user', { username: Discourse.system_user.username })
+    automation.upsert_field!('sendable_pms', 'pms', { pms: [{ title: 'A message from %%SENDER_USERNAME%%', raw: 'This is a message sent to @%%RECEIVER_USERNAME%%' }] })
+  end
+
+  context 'ran from stalled_wiki trigger' do
+    fab!(:post_creator_1) { Fabricate(:user, admin: true) }
+    fab!(:post_1) { Fabricate(:post, user: post_creator_1) }
+
+    before do
+      automation.upsert_field!('stalled_after', 'choices', { value: 'PT1H' }, target: 'trigger')
+      automation.upsert_field!('retriggered_after', 'choices', { value: 'PT1H' }, target: 'trigger')
+
+      post_1.revise(post_creator_1, { wiki: true }, { force_new_version: true, revised_at: 2.hours.ago })
+    end
+
+    it 'creates expected PM' do
+      expect {
+        Jobs::StalledWikiTracker.new.execute(nil)
+
+        post = Post.last
+        expect(post.topic.title).to eq("A message from #{Discourse.system_user.username}")
```

Comment:
Would this break if we change the translation strings?

Reply:
No that's pm template created at top of the file